packages/expo-audio/src/Audio.types.ts

```diff
@@ -441,3 +445,23 @@ export type InterruptionMode = 'mixWithOthers' | 'doNotMix' | 'duckOthers';
  * @platform android
  */
 export type InterruptionModeAndroid = 'doNotMix' | 'duckOthers';
+
+/**
+ * Recording source for android.
+ *
+ * An audio source defines both a default physical source of audio signal, and a recording configuration
```

Comment:
```suggestion
 * An audio source defines both a default physical source of audio signal, and a recording configuration.
```

Missing period at the end of the sentence.